tests/e2e/tests/regression/database/cloud-sso.e2e.ts

```diff
@@ -0,0 +1,86 @@
+import * as path from 'path';
+import { MyRedisDatabasePage, WelcomePage } from '../../../pageObjects';
+import { rte, env } from '../../../helpers/constants';
+import { DatabaseHelper } from '../../../helpers/database';
+import { commonUrl } from '../../../helpers/conf';
+import { DatabaseAPIRequests } from '../../../helpers/api/api-database';
+import { deleteRowsFromTableInDB } from '../../../helpers/database-scripts';
+import { modifyFeaturesConfigJson, refreshFeaturesTestData, updateControlNumber } from '../../../helpers/insights';
+
+const myRedisDatabasePage = new MyRedisDatabasePage();
+const databaseHelper = new DatabaseHelper();
+const databaseAPIRequests = new DatabaseAPIRequests();
+const welcomePage = new WelcomePage();
+
+const featuresConfigTable = 'features_config';
+const pathes = {
+    defaultRemote: path.join('.', 'test-data', 'features-configs', 'insights-default-remote.json'),
+    dockerConfig: path.join('.', 'test-data', 'features-configs', 'sso-docker-build.json'),
+    electronConfig: path.join('.', 'test-data', 'features-configs', 'sso-electron-build.json')
+};
+
+fixture `Cloud SSO`
+    .meta({ type: 'regression', rte: rte.standalone })
+    .page(commonUrl)
+    .beforeEach(async() => {
+        await databaseAPIRequests.deleteAllDatabasesApi();
+        await refreshFeaturesTestData();
```

Comment:
shouldn't we add addDataBase function here?

Reply:
no